.github/workflows/enzyme-ci.yml

```
@@ -49,7 +49,7 @@ jobs:
           mkdir -p build
           cd build
           rm -f config.toml
-          ../configure --enable-llvm-link-shared --enable-llvm-plugins --enable-llvm-enzyme --release-channel=nightly --enable-llvm-assertions --enable-lld --enable-option-checking --enable-ninja --disable-docs
+          ../configure --enable-llvm-link-shared --enable-llvm-enzyme --use-lld=self-contained --release-channel=nightly --enable-llvm-assertions --enable-option-checking --enable-ninja --disable-docs
```

Perhaps it needs to be quoted.
```suggestion
          ../configure --enable-llvm-link-shared --enable-llvm-enzyme --use-lld=\"self-contained\" --release-channel=nightly --enable-llvm-assertions --enable-option-checking --enable-ninja --disable-docs
```

---

Do we also want to update the readme?